packages/components/src/components/flyout-navigation/flyout-navigation/flyout-navigation-styles.ts

```diff
@@ -0,0 +1,229 @@
+        'long',
+        isSecondaryScrollerVisible ? 'in' : 'out'
+      )}, visibility 0s linear var(${cssVariableTransitionDuration}, ${
+        !isPrimaryScrollerVisible || isSecondaryScrollerVisible ? motionDurationLong : '0s'
+      })`,
+      // it's important to define background-color for each scroller to have correct scrollbar coloring
+      backgroundColor,
+      ...prefersColorSchemeDarkMediaQuery(theme, {
+        backgroundColor: backgroundColorDark,
+      }),
+      [mediaQueryEnhancedView]: {
+        visibility: 'inherit',
+        transition: 'initial',
+      },
+      // simulates frosted glass header, to be visually in sync with header of secondary scroller
+      '&::before': {
+        content: '""',
+        display: 'block',
+        position: 'sticky',
+        top: 0,
+        zIndex: 1,
+        height: frostedGlassHeaderHeight,
+        backgroundColor: frostedGlassBackgroundColor,
+        WebkitBackdropFilter: 'blur(8px)',
+        backdropFilter: 'blur(8px)', // with current frostedGlassStyle of blur(32px) scrolling becomes visually distracting
+        ...prefersColorSchemeDarkMediaQuery(theme, {
+          backgroundColor: frostedGlassBackgroundColorDark,
+        }),
+        [mediaQueryEnhancedView]: {
+          display: 'none',
+        },
+      },
+    },
+    content: {
+      display: 'flex',
+      flexDirection: 'column',
+      gap: spacingFluidXSmall,
+      padding: `${spacingFluidSmall} ${spacingFluidLarge} ${spacingFluidLarge}`,
+      [mediaQueryEnhancedView]: {
+        padding: `${spacingFluidMedium} ${spacingFluidLarge} ${spacingFluidLarge}`,
+      },
+    },
+    // header is needed to keep position of dismiss button in sync with header of secondary scroller
+    header: {
+      position: 'relative',
+      zIndex: 3, // ensures dismiss button is visible on secondary drawer in mobile view
+      gridArea: '1/1',
+      alignSelf: 'flex-start',
+      justifySelf: 'flex-end',
+      marginInlineEnd: `calc(${spacingFluidLarge} - ${spacingFluidSmall})`,
+      height: frostedGlassHeaderHeight,
+      display: 'flex',
+      alignItems: 'center',
+      [mediaQueryEnhancedView]: {
+        marginInlineEnd: 0,
+        gridArea: '1/-1',
+        placeSelf: 'flex-start',
+      },
+    },
+    dismiss: {
+      padding: spacingFluidSmall,
+      [mediaQueryEnhancedView]: {
+        margin: spacingFluidSmall,
+        padding: spacingStaticSmall,
+      },
+    },
+  });
+};
+
+// TODO: getBackdropJssStyles can be shared with flyout and modal
+/**
+ * Generates JSS styles for a frosted glass background.
+ * @param {boolean} isVisible - Determines if the frosted glass effect is visible.
+ * @param {string} duration - The duration of the transition animation.
+ * @param {Theme} theme - The theme to be used
+ * @returns {JssStyle} - The JSS styles for the frosted glass backdrop.
+ */
+const getBackdropJssStyles = (isVisible: boolean, duration: MotionDurationKey, theme: Theme): JssStyle => {
```

Comment:
`JssStyles` with plural `s` or `JssStyle` to match the type? Probably a consistency thing across all similar utils.